src/control/server/ctl_storage_rpc.go

```diff
@@ -170,10 +171,31 @@ func (c *ControlService) scanScm(ctx context.Context, req *ctlpb.ScanScmReq) (*c
 	return newScanScmResp(c.getScmUsage(ssr))
 }
 
+// Check if a PCI is a VMD address
+func isVmdAddress(addr string) bool {
+	subAddrs := strings.Split(addr, ":")
+	return "0000" != subAddrs[0] && subAddrs[2] == "00.0"
+}
+
+// Convert a VMD address into a PCI address
+func vmdToPci(vmdAddr string) string {
```

Comment:
Please use the existing function for this as well

Reply:
For sure, I was not aware that such function was existing.

- [ ] Use VMD dedicated functions